[Docs] Standardization of the end of each documentation page - Get Started - Erik Review (#14774)
unoplatform/uno: doc/articles/get-started-explore.md

## Next Steps

> Now that you have built your first application and explored the Uno Platform, learn more about:

Reviewer: ```suggestion
Now that you've built your first app and explored the Uno Platform, learn more about:
```
